### ISSUE TITLE
[live preview] problem with certain expressions

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-website#57]
From https://github.com/mathjax/MathJax/issues/1480

``` html
<math>
    <mmultiscripts>
        <mmultiscripts>
            <mi>A</mi>
            <mi>B</mi>
            <mprescripts />
            <none />
            <mi>D</mi>
        </mmultiscripts>
        <mrow>
            <mo>(</mo>
            <msup>
                <mi>g</mi>
                <mi>h</mi>
            </msup>
            <mo>)</mo>
        </mrow>
        <mprescripts />
        <none />
        <mi>i</mi>
    </mmultiscripts>
</math>
```

throws errors in the live demo.

**Comment by dpvc** [comment]
This isn't a website issue it is a MathJax CHTML issue.  I have solved it in the original mathjax/MathJax#1480 issue tracker.

**Comment by pkra** [comment]
Thanks!
